packages/maskbook/src/database/Plugin/index.ts

```diff
@@ -59,3 +59,11 @@ export const createPluginDBAccess = db
 export function toStore(plugin_id: string, value: unknown): InStore {
     return { plugin_id, value }
 }
+
+export interface Plugin {
```

Comment:
```suggestion
export interface PluginMetadata {
```

plz do not use too short identify.

Reply:
Just wanted to follow naming style of `Persona` in `PersonaCard`.

Reply:
Updated, please take a look.